heidrun/hathi_marcxml.rb

```diff
@@ -0,0 +1,350 @@
+contributor_map = lambda { |r|
+  df_700 = Heidrun::MappingTools::MARC.datafield_els(r, '700')
+  df_710 = Heidrun::MappingTools::MARC.datafield_els(r, '710')
+  df_711 = Heidrun::MappingTools::MARC.datafield_els(r, '711')
+  df_720 = Heidrun::MappingTools::MARC.datafield_els(r, '720')
+
+  datafields = []
+  datafields << df_700
+  datafields << df_710
+  datafields << df_711
+
+  if (Heidrun::MappingTools::MARC.subfield_values(df_720, 'e') & ['joint author', 'jt author']).empty?
```

Comment:
I'm just flagging some lines that are longer than 80 characters. :cop: :smile:  A few more below...

(We admittedly have some comment lines in the Florida mapping that are > 80 that should be wrapped.  Don't worry about anything that can't be conveniently broken, like long URLs.)